Sources/NIO/Socket.swift

```diff
@@ -174,14 +174,27 @@ typealias IOVector = iovec
 
         return try withUnsafeHandle { handle in
             return try withUnsafeMutablePointer(to: &vec) { vecPtr in
+#if os(Windows)
+                var messageHeader =
+                    WSAMSG(name: notConstCorrectDestinationPtr
+                                    .assumingMemoryBound(to: SOCKADDR.self),
```

Comment:
For my own OCD, can we `.assumingMemoryBound(to: sockaddr.self)`? I'm pretty sure they typealias to the same thing, but this way I can see by inspection that the assumption is right.